src/AsmResolver.Workspaces.DotNet/Analyzers/Definition/MethodAnalyzer.cs

```diff
@@ -46,7 +46,7 @@ private static void ScheduleMembersForAnalysis(AnalysisContext context, MethodDe
             }
 
             // Schedule method body for analysis.
-            if (subject.CilMethodBody is not null && context.HasAnalyzers(typeof(CilMethodBody)))
+            if (subject.IsIL && subject.CilMethodBody is not null && context.HasAnalyzers(typeof(CilMethodBody)))
```

Comment:
Why is this extra check required? If the method body is a CIL body, then `CilMethodBody` is not `null`.

Reply:
CilMethodBody can throw exception when body is native

Reply:
This sounds like a bug to me then. We'll leave it in this PR for now.